exporter.go

```diff
@@ -196,36 +198,12 @@ func parseUnixSocketAddress(address string) (string, string, error) {
 	return unixSocketPath, requestPath, nil
 }
 
-func getListener(listenAddress string) (net.Listener, error) {
-	var listener net.Listener
-	var err error
-
-	if strings.HasPrefix(listenAddress, "unix:") {
```

Comment:
We want to keep the option to use a unix socket. You can still use this logic in combination with the TLS config

Reply:
see answer below